Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -298,6 +298,24 @@ def condition_(self, fragment, stepList, key, hasHarvest):
     '--procModifiers': 'trackingMkFit'
 }
 
+# Vector Hits workflows
+class UpgradeWorkflow_vectorHits(UpgradeWorkflow):
+    def setup_(self, step, stepName, stepDict, k, properties):
+        stepDict[stepName][k] = merge([{'--procModifiers': 'vectorHits'}, stepDict[step][k]])
+    def condition(self, fragment, stepList, key, hasHarvest):
+        return fragment=="TTbar_14TeV" and '2026' in key
+upgradeWFs['vectorHits'] = UpgradeWorkflow_vectorHits(
+    steps = [
+        'Reco',
+    ],
+    PU = [
+        'Reco',
+    ],
+    suffix = '_vectorHits',
+    offset = 0.9,
+)
```

Comment:
@kpedro88 
something is missing here, apparently.
With this PR I do not see any `--procModifiers vectorHits`  in `runTheMatrix.py -w upgrade -n -e`

Reply:
just as a sanity check, it's not just me, the cms-bot/jenkins cmdLog for the .9 workflow does not have it either 
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-7ffe9a/9899/runTheMatrix-results/23234.9_TTbar_14TeV+2026D49_vectorHits+TTbar_14TeV_TuneCP5_GenSimHLBeamSpot14+DigiTrigger+RecoGlobal+HARVESTGlobal/cmdLog

Reply:
I must have broken it when doing the last rebase as there were conflicts in this file. Working to fix it...

Reply:
Phase2 workflows use the `RecoGlobal` step, not the `Reco` step.

Reply:
That fixes it, thanks! That probably also means that it never actually worked in the first place, sorry!